src/gateway/understanding-kong/key-concepts/services.md

```diff
@@ -1,6 +1,33 @@
 ---
 title: Services
-
+content_type: explanation
 ---
 
-## PLACEHOLDER Services
+In {{site.base_gateway}}, a service is an entity representing an external upstream API or microservice. For example, a data transformation microservice, a billing API, and so on. 
+
+The main attribute of a service is its URL, where Kong should proxy traffic to. You can specify the URL with a single string, or by specifying its protocol, host, port, and path individually. 
+
+Service entities are abstractions of each of your own upstream services. Examples of Services would be a data transformation microservice, a billing API, etc.
+
+## Service configuration
```

Comment:
I think you can put this section last. Or fold the two bullet points into the first paragraph.